cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js

```diff
@@ -74,6 +77,37 @@ export default {
       });
   },
 
+  searchFieldMappingProfile(text) {
+    cy.do(searchField.fillIn(text));
+  },
+
+  verifySearchButtonEnabled: () => cy.expect(searchButton.has({ disabled: false })),
+  verifySearchButtonDisabled: () => cy.expect(searchButton.has({ disabled: true })),
+
+  verifyClearSearchButtonExists: () => cy.expect(clearButton.exists()),
+  verifyClearSearchButtonAbsent: () => cy.expect(clearButton.absent()),
+
+  clearSearchField: () => {
+    cy.do(searchField.focus());
+    cy.do(clearButton.click());
```

Comment:
use array, like this 
    cy.do([
      searchField.focus(),
      clearButton.click()
    ]);

Reply:
fixed